casbin/dllmain.cpp

```diff
@@ -1,35 +0,0 @@
-/*

```

Comment:
You deleted the DLL main function for Windows. Will Windows build still work?

Reply:
Yes

Reply:
The windows.h is a utility library related to windows specific implementation of C++. We don't have such specific dependencies in our application

Reply:
They are actually redundant files.

Reply:
So where is the DLL entry now?

Reply:
The type of library which we are using is a static library a .lib file for exporting this project and making use of this project in any other MVS C++ project. Hence, we don't require a DLL entry.